storage/README.md

```diff
@@ -7,6 +7,8 @@ Introduction
 ------------
 
 - [Read more about Cloud Storage for Firebase](https://firebase.google.com/docs/storage)
+- Note that the APIs used in the Swift example involving the Swift Result type
```

Comment:
Nit: I tend to write "Note:" in comments all the time too, but technically this file is full of notes, so consider rephrasing this.